bindings/src/client/entities/Player.js

```diff
@@ -1127,3 +1096,21 @@ alt.on('playerStartTalking', (player) => {
 alt.on('playerStopTalking', (player) => {
     mp.events.dispatchLocal('playerStopTalking', player.mp);
 });
+
+
+alt.on('enteringVehicle', (vehicle, seat) => {
+    mp.events.dispatchLocal('playerStartEnterVehicle', vehicle?.mp, altSeatToClientMp(seat));
+});
+
+alt.on('enteredVehicle', (vehicle, seat) => {
+    mp.events.dispatchLocal('playerEnterVehicle', vehicle?.mp, altSeatToClientMp(seat));
+});
+
+alt.on('changedVehicleSeat', (vehicle, oldSeat, seat) => {
+    mp.events.dispatchLocal('playerEnterVehicle', vehicle?.mp, altSeatToClientMp(seat));
+});
```

Comment:
if its only triggered when player enters driver seat, then there should be a check for it
```suggestion
alt.on('changedVehicleSeat', (vehicle, oldSeat, seat) => {
    // not driver? ignore
    if (seat !== 1) return;
    mp.events.dispatchLocal('playerEnterVehicle', vehicle?.mp, altSeatToClientMp(seat));
});
```